File: jobtrackpro-frontend/src/services/reminderService.js
import api from "../api/api"

export const createReminder = (reminder) => {
    return api.post(`/reminders`, reminder);
}
export const getReminders   = () => {
    return api.get(`/reminders`);
}

export const deleteReminder = (reminderId) => {
    return api.delete(`/reminders/${reminderId}`);
}

File: jobtrackpro-backend/src/main/java/com/jobtrackpro/service/ReminderService.java
package com.jobtrackpro.service;

import java.util.List;

import com.jobtrackpro.model.Reminder;

public interface ReminderService {
    
    Reminder createReminder(Long userId, Reminder reminder);

    List<Reminder> getReminderByUser(Long userId);

    void deleteReminder(Long reminderId);
}
